Index.js

```diff
@@ -0,0 +1,58 @@
+const list = document.getElementById('lists');
+const form = document.querySelector('form');
+const author = document.getElementById('atr');
+const title = document.getElementById('ttl');
+
+const bookla = [];
+
+function collection(title, author) {
+  this.title = title;
+  this.author = author;
+}
+
+function alocalStorage() {
+  const key = title.value;
+  localStorage.setItem(key, JSON.stringify(bookla));
+}
+
+function newCollection() {
+  // eslint-disable-next-line new-cap
+  const nCL = new collection(title.value, author.value);
+  bookla.push(nCL);
+  alocalStorage();
+}
+
+form.addEventListener('submit', newCollection);
+
+function clOutput() {
```

Comment:
For better practices try to use descriptive names for your variables, and functions so that other developers can read your code easily